docs/source/conf.py

```diff
@@ -47,7 +47,7 @@
 autosummary_generate = True
 autoclass_content="class"
 autodoc_default_options = {
-    "members": True,  # Include all methods.
+    #"members": True,  # Include all methods.
```

Comment:
Probably set it to False or just remove it?

Reply:
Will remove.